code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -29,7 +29,7 @@
 
 /obj/item/gun/energy/pulse/carbine
 	name = "pulse carbine"
-	desc = "A next-generation pulse weapon for Nanotrasen's security forces. High production costs and logistical issues have limited its deployment to specialist Loss Prevention and Emergency Response units."
+	desc = "A next-generation pulse carbine capable of decimating overwhelming opposition. This weapon has accrued a kill count of 2564 dead since it was first seen in issue #10 'DEATHSQUAD CLASSICS'."
```

Comment:
an exact kill count will probably be outdated real fast

Reply:
can you make it change a var and said var is displayed in the description

Reply:
> can you make it change a var and said var is displayed in the description

No because it's talking about the kc in the comics. But I'll change this.

Reply:
changed